README.md

```
@@ -12,25 +12,25 @@ cd playtak-ui
-Running locally with NodeJs requires (version 14 or later) and NPM. [Node](http://nodejs.org/) and [NPM](https://npmjs.org/) which are really easy to install.
+Running locally with NodeJs requires (version 14 or later) and NPM. [Node](http://nodejs.org/) and [pnpm](https://pnpm.io/) which are really easy to install.
 
 To make sure you have them available on your machine, try running the following command.
 
 ```sh
-node -v && npm -v
+node -v && pnpm -v
 ```
 
 To get started run the following:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)